src/screens/styles.ts

```diff
@@ -4,7 +4,103 @@ export const styles = StyleSheet.create({
   container: {
     flex: 1,
     flexDirection: 'column',
-    alignItems: 'center',
     backgroundColor: 'white',
+    padding: 10,
+  },
+  profileHeader: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    marginBottom: 8,
+    justifyContent: 'space-between',
+  },
+  profileName: {
+    fontWeight: 'bold',
+    fontSize: 16,
+    marginLeft: 8,
+  },
+  postDate: {
+    color: 'gray',
+    paddingLeft: 120,
```

Comment:
You should be able to create an offset like this using `justifyContent` and `alignItems` attributes.